packages/terra-framework-docs/src/terra-dev-site/test/embedded-content-consumer/consumers/ErrorConsumer.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Consumer } from 'xfc';
+import classNames from 'classnames/bind';
+import EmbeddedContentConsumer from 'terra-embedded-content-consumer';
+import styles from './Consumer.module.scss';
+
+const cx = classNames.bind(styles);
+
+Consumer.init();
+
+const ErrorConsumer = () => (
+  <EmbeddedContentConsumer
+    className={cx('iframe')}
+    options={{ iframeAttrs: { title: 'Error Consumer' } }}
+    src="https://www.google.com"
+  />
+);
```

Comment:
nit: could we add a comment or rename it to include the reason that this one is an error consumer?